src/components/questionStructure/Task/variants/HandwritingTask/types.ts

```diff
@@ -0,0 +1,10 @@
+import { ExcalidrawElement } from '@excalidraw/excalidraw/types/element/types'
+import { AppState } from '@excalidraw/excalidraw/types/types'
+
+export interface HandwritingAnswer {
+  excalidraw: {
```

Comment:
How about concealing the implementation detail here and calling this something like

```suggestion
  raw: {
```

instead? The word `excalidraw` makes little sense in the API's DB, if you think about it.

Reply:
Sounds reasonable to me. Addressed in 08caaa90893cd0a7af53f4af047093308d09c5fe.